docs/develop/convert-xml-to-json-manifest.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Convert an add-in to use the unified manifest for Microsoft 365
+description: Learn the various methods for converting an add-in with an XML manifest to the unified manifest for Microsoft 365 and sideload the add-in.
+ms.topic: how-to
+ms.date: 06/08/2023
+ms.localizationpriority: medium
+---
+
+# Convert an add-in to use the unified manifest for Microsoft 365
+
+There are three basic tasks to converting an add-in project from the XML manifest to the unified manifest for Microsoft 365:
+
+- Ensure that you have 64x64 pixel and 128x128 pixel images files to serve as icons for the add-in.
+- Convert the XML manifest itself to the JSON format of the unified manifest.
+- Package the new manifest and main icon and high resolution icon image files into a zip file for sideloading or deployment.
+
+> [!NOTE]
+> 
+> - The unified manifest is a preview feature for Office Add-ins and is currently supported only for Outlook.
+> - Projects created in Visual Studio, as distinct from Visual Studio Code, can't be converted at this time.
+> - If you [created the project with Teams Toolkit](teams-toolkit-overview.md) or with the "unified manifest" option in the [Office Yeoman Generator](yeoman-generator-overview.md), it already uses the unified manifest, so you can ignore this article.
+
+## Ensure that you have the two image files
+
+It's highly likely that you have the two image files already in your add-in. They are specified in the [IconUrl](/javascript/api/manifest/iconurl) and [HighResolutionIconUrl](/javascript/api/manifest/highresolutioniconurl) elements in the XML manifest. But these are both optional elements, so it's possible your add-in project doesn't have one or both. These are required for an add-in that uses the unified manifest, so you need to create them if the project doesn't already have them. For guidance, see [Create an icon for your add-in](/office/dev/store/create-effective-office-store-listings#create-an-icon-for-your-add-in). 
```

Comment:
```suggestion
It's highly likely that you have the two image files already in your add-in. They are specified in the [IconUrl](/javascript/api/manifest/iconurl) and [HighResolutionIconUrl](/javascript/api/manifest/highresolutioniconurl) elements in the XML manifest. But these are both optional elements, so it's possible your add-in project doesn't have one or both. These are required for an add-in that uses the unified manifest, so you need to create them if the project doesn't already have them. For guidance, see [Create an icon for your add-in](/office/dev/store/create-effective-office-store-listings#create-an-icon-for-your-add-in). 
```
```suggestion
It's highly likely that you have the two image files already in your add-in. They're specified in the [IconUrl](/javascript/api/manifest/iconurl) and [HighResolutionIconUrl](/javascript/api/manifest/highresolutioniconurl) elements in the XML manifest. Because these are both optional elements, your add-in project mignt not have one or both. These are required for an add-in that uses the unified manifest, so you need to create them if the project doesn't already have them. For guidance, see [Create an icon for your add-in](/office/dev/store/create-effective-office-store-listings#create-an-icon-for-your-add-in). 
```